src/components/ec-dropdown-search/ec-dropdown-search.spec.js

```diff
@@ -289,6 +290,19 @@ describe('EcDropdownSearch', () => {
     expect(wrapper.findByDataTest('my-test-list').element).toMatchSnapshot();
   });
 
+  it('should initialize the "useFocusTrap composable" with mandatory options', () => {
+    mountDropdownSearch({ }, {
+      slots: {
+        default: '<a href="#" @click.prevent>Open me</a>',
+      },
+    });
+    const options = useFocusTrap.mock.calls[0][1];
+    expect(Boolean(options.clickOutsideDeactivates)).toBe(true);
```

Comment:
Why do you convert these to `Boolean`? They are all boolean already.

Reply:
corrected, thanks